apps/web-giddh/src/app/invoice/invoice.component.ts

```diff
@@ -35,6 +37,13 @@ export class InvoiceComponent implements OnInit, OnDestroy, AfterViewInit {
         ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
             this.isMobileView = result.matches;
         });
+
+        this._breakPointObservar.observe([
```

Comment:
Avoid creating redundant subscriptions. This subscription is already present in constructor, club them as discussed